### PR TITLE
create account api-doc update

### PR DESCRIPTION
**Pull request by gshannon-immersve**
update to add "version" and "liveness" variables to response schema within api-docs for create account.

### CODE REVIEW

File: imsv-docs-docusaurus/openapi/endpoints/accounts/models/account.yaml [hunk 1]
```diff
@@ -4,6 +4,10 @@ properties:
     type: string
     description: The unique account ID
     example: e4b66d1ddecbf518f5f337n5e5a4f682
+  version:
+    type: string
+    description: response schema version
```
Comment (ncjones) [inline imsv-docs-docusaurus/openapi/endpoints/accounts/models/account.yaml:9]:
```suggestion
    description: The resource version number.
```

File: imsv-docs-docusaurus/openapi/endpoints/accounts/models/account.yaml [hunk 2]
```diff
@@ -27,3 +31,9 @@ properties:
       - cardholder
       - identity
       - partner
+  liveness:
+    type: string
+    description: indicates status as either test or live
```
Comment (ncjones) [inline imsv-docs-docusaurus/openapi/endpoints/accounts/models/account.yaml:36]:
```suggestion
    description: The resourc liveness mode: "test" or "live".
```

**Review by ncjones — APPROVED** [review]

File: imsv-docs-docusaurus/openapi/endpoints/accounts/models/account.yaml [hunk 3]
```diff
@@ -27,3 +31,9 @@ properties:
       - cardholder
       - identity
       - partner
+  liveness:
+    type: string
+    description: The resourc liveness mode: "test" or "live".
```
Comment (ncjones) [inline imsv-docs-docusaurus/openapi/endpoints/accounts/models/account.yaml:36]:
```suggestion
    description: The resource liveness mode: "test" or "live".
```

**Review by ncjones — APPROVED** [review]

File: imsv-docs-docusaurus/openapi/endpoints/accounts/models/account.yaml [hunk 4]
```diff
@@ -27,3 +31,9 @@ properties:
       - cardholder
       - identity
       - partner
+  liveness:
+    type: string
+    description: The resource liveness mode: "test" or "live".
```
Comment (braden-holgate) [inline imsv-docs-docusaurus/openapi/endpoints/accounts/models/account.yaml:36]: isn't the test or live in the description redundant?

**Review by maticko — APPROVED** [review]

**Pull request merged**